frontends/web/src/locales/en/app.json

```diff
@@ -72,7 +72,7 @@
     "signing": "To proceed, confirm message on your BitBox02.",
     "success": {
       "message": "Proceed on {{host}}",
-      "title": "Message successfully confirmed and sent"
+      "title": "Address request confirmed and sent"
```

Comment:
Better but I would still just do `Address successfully sent` (it is not the address request that is sent).

Reply:
👍    changed in https://github.com/digitalbitbox/bitbox-wallet-app/compare/488fafe2a809ff7687c60473e0f8f086105418f9..d958cc53580c66825c8208c4fbb95530c7d53df1